### DISCUSSION TITLE
[OPINION] It is plagued with pop ups to participane in a survay.

### DISCUSSION THREAD

**Discussion by MerryPanda** [phcode-dev/phoenix#1535] [General]
Pls disable all unneeded pop ups (survey pop ups in particular). It is a horrible user experience.

**Comment by abose** [comment]
@MerryPanda thanks for the suggestion and i too agree that the popups are a bit more than what even i would like.

The survey popup only comes once in the lifetime of using the product. All the other popups are feature awareness popups which unfortunately, we had to place in as we found that those feature usages were not known to any users, and once we added the popups, the feature usages pretty much jumped to above 25% users.

Each of the popups were statistically measured and placed, the underlying stats was in favor of the popups. I understand that it is annoying for long term Brackets users.

BTW all the feature popups are only shown once, so, you will not see that again. I will try to see if i can change the placement or timing of these popups, or maybe an expert mode that will not have any popups on first install. 

What would be your suggestion for feature awareness without the popups. will and expert mode help? or should all the popups come in one go?

**Reply by MerryPanda** [reply]
Thanks for the responce.

I think that it will be better to have a "proficiency" option which would control the need for all of those pop ups. Which is selected on start up or installation.

Or maybe make a link to the latest news (like in Chrome) - so there will be no need for pop ups at all.

**Reply by abose** [reply]
This is being addressed @ https://github.com/phcode-dev/phoenix/pull/1537
